tests/test_physics/test_triples.py

```diff
@@ -48,39 +48,39 @@ def test_vacuum_state_Abc(self, n_modes):
         A, b, c = triples.vacuum_state_Abc(n_modes)
 
         assert np.allclose(A, np.zeros((n_modes, n_modes)))
-        assert np.allclose(b, np.zeros((n_modes)))
+        assert np.allclose(b, np.zeros(n_modes))
```

Comment:
I think it'd be best to use `np.all` instead of `np.allclose`

Reply:
Working on it